Themes/default/languages/ManageSmileys.english.php

```diff
@@ -93,8 +93,7 @@
 $txt['icons_new_icon'] = 'New message icon';
 $txt['icons_location_first_icon'] = 'As first icon';
 $txt['icons_location_after'] = 'After';
-$txt['icons_filename_all_gif'] = 'All files must be &quot;gif&quot; files';
-$txt['icons_filename_all_png'] = 'All files must be &quot;png&quot; files';
+$txt['icons_filename_all_png'] = 'Files must be of type "png"';
```

Comment:
Then this phrasing is misleading, no? The *extension* must be PNG, not the "type" (which could refer to either the image type or the extension).

Reply:
Yes, you are right. I will change